udata/templates/embed-dataset.html

```diff
@@ -5,7 +5,7 @@
     }
     .dataset-card {
         border: 1px solid #EEE;
-        min-width: 300px;
+        min-width: 0;
```

Comment:
Thanks for [the link](https://hackernoon.com/11-things-i-learned-reading-the-flexbox-spec-5f0c799c776b) in the commit message. 👍